From cae50ae9cdc18a845b5f95c0ef8b67ae8e88c55e Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 29 Oct 2026 09:06:52 +0100
Subject: [PATCH 1/2] Update the upstream signing key

Refresh ntpsec.gpg.pub.asc from keyservers/WKD: same certificate(s),
with self-signatures our copy was missing.
The commands are in the comment above its Source line.

The current key is needed to verify the release signature with
openpgpverify (sqv), which checks that the key was valid when the
release was signed.

Assisted-by: Claude Opus 5.5
---
 ntpsec.gpg.pub.asc | 316 ++++++++++++++++++++++++++++-----------------
 ntpsec.spec        |   4 +
 2 files changed, 199 insertions(+), 121 deletions(-)

diff --git a/ntpsec.gpg.pub.asc b/ntpsec.gpg.pub.asc
index 8d3661e..476bbb3 100644
--- a/ntpsec.gpg.pub.asc
+++ b/ntpsec.gpg.pub.asc
@@ -1,6 +1,6 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQINBFcPDLMBEACenOnM3H3AGmqlVD2PUW8mH4SmtkPFyCDg4UiWu7TQCO86W7ha
+xsFNBFcPDLMBEACenOnM3H3AGmqlVD2PUW8mH4SmtkPFyCDg4UiWu7TQCO86W7ha
 3H4tA77UpEpbfoCh3RrScgEDhQpFpjfKjxq3pg+nbPZlfId3dWsk01QeyE/JYL7A
 eUmDsR2trjnhVqjXAHHzEi2G7Mvf/xfuwvJaFN9f7l0rpm3kypuE1hsEByo4qydj
 vXkL3MEp08j3Id+fQK3sJZCfj7wyvjE6URC8UJYxMkf/lrxBWpiDTAfTrrYGofdu
@@ -11,124 +11,198 @@ ocDggJ5oKaNqd8rKgzKFk+qNXQnTpzyI4QKnHu48AKOdeJ+3Wsuhq4Nm/foSLxe3
 kvXdLLvOhgsPyvnaAsgM0jw8z2ugc9TdhxEchBnP1sw6nZKZ72a9HPSReashrBiD
 VTDM2yjHPP+0i6Cgl6iZLKTbl4pcLVj4uDgQIG+PNBfUPjW4CJjrFv7sBevRRSHi
 SyODtvZzxnqCpRFG2cGkY6OQEdhZwEJIKHXjOkxZ0WOwh3k3OnS2pwKbIwARAQAB
-tC9OVFBzZWMgU2VjdXJpdHkgUmVwb3J0aW5nIDxzZWN1cml0eUBudHBzZWMub3Jn
-PokCVgQTAQgAQAIbAwcLCQgHAwIBBhUIAgkKCwQWAgMBAh4BAheAFiEE2j/fd0zH
-D6ZHKexFBdmzcUd8dSgFAl9757wFCRs43gkACgkQBdmzcUd8dSjJRBAAnpoiqeT/
-D64HUOEPAlc5apNgMnGt6PFRWIrUTN9v2cXeDJWQ7DCIdlqV0rhHwRT3WL6aZY5i
-igu1FTun8bsV9hSCCQ7BveTIZFAMg+4bsjDBANNvteT/329Am1cSrlGzK6U5HVHU
-nNZhP2PrVsqg9OVIF/u3lkO8AJjYBvp8Jlvvbl2MOcojAR7DqTslmCZaacjA9zuI
-JJ6L+V56KU+l9xvO6A8XOoi8xPdxS11seZiMRODuKYC4AAdkBmcNKZVuOWqTQfxn
-txEkaI5Q3t+kivkZiclqYVojkVCm77IQI3+23w1YikW1EJJ6lf9MNzrq9DMG0A4B
-hopZ8mslazvOBMydq9wtpbXL7bZV3+a9wV+KQdIQhb0go7DaSfOxzW/+QyTL8dCd
-khmip4o/eOLuteoIBODHZE84vBU8zCZACbgjE4nSGNIit091PBN91izhWUvBD7FX
-tra69i4JoRW2sV3KZy1zsBMJ+ptULtfHD1A46Ss8fEYaRgZASXvmSrpH0hL8uA2O
-OxcHb5Vo8+O4Ofx47SrD7Lf06cw/pIfd5acQ/Td3pJj8SxYECqVtjOIuusOVawEx
-MXtaSMTQagoGmc16O7yvASibnArQxKbX6LACzEV5o3V12ZS3HT6yVSU3MN3xgsdb
-pA7O1bPEwz4zsQIhI+DvHBVlvDD0yikTWluJAj8EEwEIACkFAlcPDLMCGwMFCQWj
-moAHCwkIBwMCAQYVCAIJCgsEFgIDAQIeAQIXgAAKCRAF2bNxR3x1KCISEACHp8Ut
-Rx2ZRQ3KyBQnNF6k+RD4T9k4dQPomve8P3rWKEY373Q7jMAFp+UGerNgTIyGpus0
-UKzswrLyeSOT3lvA5nU66r95IPqpWeYBy4o1lgSDq/xdAINkRozw2CpzsUdg+APU
-WbiddKfqlcDaHOfyWT9Vo83YLX+o1FZkWE5+7Y2hIfiI5cuLYpRhCoOGVJlSHBOR
-0ihIlF9n1PACo+dd6q4++Kyw47I+OCGx2qxuWS25CrxWEupjSivQvw8DuD9VWQMa
-03sUKUUQ9MSDrt+soAUYrsShttAfy6UKTIY+inDA+e6b150qzJZ4PbMsChNAHhUA
-76lHqO1mrAISNi+vbMnXOshGQrs70svbgxYibHi636eyn1yivQND8nadVPbQQwPX
-6plKbqup4XVntT7idH5ZA99Ny7WXpRzbJ0yL/Jc1YRrZ1t5QeNYrhLld/IjN1rsp
-MS0OlQeIGMvidhCR8DEsDjQJ405QAwgwiAaum5DBiiQ12U8eAqgpF89OVca+Q3bt
-15mCNUkBF4fj7hJwbKRtqmHBi8b6I3x+dXqcl0hma6M+n7wc9sbN/fR/m0OHSV/U
-66ywRKu47199p3Yj0etSP39tgWxiUdhvx0tdbOeGA1RygXNjLTsTOa5EBHuKetWW
-4WAOgM2uYqgotTAUNjalFZbkz2AM94hMQWGrqokCHAQQAQgABgUCVw8NmgAKCRBz
-IpFyYszB5d+2EACbTkI7XkF7wBI7pzEf8avOhwg0qKA3slX7oYqrnkaaCSMISllF
-m7U2tPTOJJQOtno4NfmURcvdMg3ozyYUeeBj+yUEYhM5PGm9j47pzcCkf8/2R9Mp
-Z4oOCo9pKOuoWATG2wVMZEGh5aNs3Sq/f11ao9RSYEv1pxgy8dFBUQ+43v7WBImC
-WyU/jHWmPqewLOADzESZRriMGfaSXs1BbCfggmiV+EHe1RhbXCzdPDCIQugDTg+Z
-9kHrn44TTKXJJ3Fhord+klIwO06iR7poz8EO4ZiDqK1t2QBwtVAsUPd/RbO0lSuG
-QJ0OYDHF/6W3DJn1leGZ+4ozDWmrQEObt8ciVOYf+qPws/4qEdsw5E2r3ZsofBYC
-j4zqqJntZm4+3d98zpLIbZ/KdUAKPuDLZzxPbcvPXDM5RNUvFnnEjz6NJMcRP2CS
-UBxugndmqGQm5xXZ8DBNcfH73WtHyWedlxUaS/uY4VzvW+nA5OJLV/BvDzKTwdq4
-T4IrGX2Fevg/Sl8PUWHeUlNyFMBa6c3gB90MjkznQFwPTX9WsEXkAIYSJSEdg+aH
-nhGiQXAuETTqaEEiW76jtS6l8cJ5aFNXQa4GW5cZckJTlUHvkv8RTrTW8BUjgeAq
-azzQbTYDnhwEJms1pPASHPF4LXpmauffJ4qcKAsPTIfYSFptMMxRxn2JKokCHAQQ
-AQgABgUCVw8OCwAKCRAqfD42zCgtvsLKD/9ssv6W4gj9O5DZf+ISfo+7AdrwmjN8
-ppvhy/699I3duGevKKQS7UEHY+Sjmf9YVWRzI1AGvJsjUsb/Btfa8zNijtP3PH6v
-eTtlFOVeNP6OAjcUVgePBgHgMqu17m+EGGReH7a8ruBl7Z6wzIU60qHX27hPbZHm
-12W6X+EF7/+QevVurVdVYG3gzFM7J7Zbh05RWq0JUVimRk5zadrHRam+5yxPGsx1
-Hf5lzR2XIRoFxNzBh8eN9ilpbQ37GidTEyD4fOt3MKIEBGr1pFBnFc+ivxBsAMAI
-7a2bkku1v15d84zyjEAAgE17/AmYtqH2EIzMZ7I8a/sOIdLfaULT8uxr73yivC7Y
-nGMPamL4KMmo0XHj/TjXop4nWIP4rma8G+yPYlk8PMpEHVyQZLcG1mtFmQKMtWSe
-jiZfVNtcYzAVbrC1t0wF9b5x8VfJACVwV8JkGV8CwfabqEzouIq1dh9Sm793bwG4
-LKd+W0DP5+QSYacpCEkgPeq4jVVZ+SEN+bCstEJwxW1WJ1p3rIBIGESpX+NKFAGl
-Hl8i1fhrJ1foZr9Gx4L94GKrU4zgmB457I/t1k9iRpYVDfICgY5sy6ZHcrkhLiOZ
-hr199QWHv33Bd7u3Y5pqoLamA3bjPZEJk82p7ywMas0z4sg2kdzol7iC5OjBjySv
-gRFclkrSfKK9VrkCDQRXDwyzARAA0XTbXJBhJMaAOFvPWaw+dThy4RDlvV3hHiNJ
-9uIIROYnmiu4W6t+jrZsJ3WWdJqNggY+XLnwVyRXWbmFXf8qViStoBfs62cDzmYn
-dsfnl5ZcRdqrHJ+ErdblYU1Sc2WdnfoJQY49VaSLNJMDvU9gRdC9dVdI9Z2YR+m7
-RNXNZbqtxdDMgMFZYBq3jdWNkutpQA9PKwco4qGpQdsWLpyGXZRHF6iz63WRWhAe
-YhLftUvoTXCISN+WTSuT4tC0pDfDfcdX7ber5icVg7Uh4aizFTenS4jlsbkE1I93
-oZCaPN3t8gah+iiCkIRVYpd0jy6RpZXIHLvRrgmdpZZ59Bkg3Xi4WtggJ5jg0wc8
-csC545M0wXlgktQN8w9RibR0woJoPUkh/dWiyXU/eEsQ84G1iMBHUGBDJ9g/vu/d
-nqhRiu/A+jx2JbeYuxvR1MmL6WXJmQ8eHdJvnUQiS9nRqb8xQeSxoZlrBNw1QULk
-11fFMy0h3adOX3KXbnqZsPUvCkt5P3UyEJSQ4lnN5K/Acbd/5toENeB2kq1hLdJk
-HthrblnfkNtefY1fPHKtp56UqsuUcaLUpvLijUntR4PSwAbhDNKXKdLJroSNV0Wr
-rOH1Rn64apQlrPw0ZwUdIdjXQFCxdMsagL2TgwuULmYjQUP/noX0c4bmglqmcKel
-KrROdCEAEQEAAYkCNgQYAQgAIAIbDBYhBNo/33dMxw+mRynsRQXZs3FHfHUoBQJf
-e+dGAAoJEAXZs3FHfHUoEq8P/35SlCVQIwWv9x7Bg3rpJR1WO5VDX//+UWWP5i2+
-fF8LGgkwNNno63VSSsJmla8A6IAT5pAisixCJuXJrQ2RWUiVgCOvUIOgczQFuv83
-T+mnbFh6Y9Ic9Ag4p5sLRDiwofceB+lNIOlmjAdg1KhFbBEczyRWqho5khfKSsxM
-OtJ1LhcGwEhWF3DeHjJgGdb1SuAwpxiLpa/DgASLYXhEFfayYxlT4la+SkpGJXzK
-gEZ7vc5GB0FKAD0/Jz9Yq1fBoR0j6pyEhqa4p1okUTP8YxsNHmw9gUcwLqs44JS/
-AhTJ1pghdha0m3h4pUV7k1LpqXbCqKPl+edwNLotN9I7WlfcEJuBAudP46f9mq29
-N0cp3ATUmuZsioPkr+u+LDK7uSylAJaJ3wG+RSGjTiV4N/AlN6W4syT8PAT/cfjW
-sdboKD0Df8lVx/PUHsmeLc1vfNoawo7KwjNVaksBiXvGrXgMhTs3Koub5rMpyMB8
-OXsFGPwho62yr6euMW1zrxmlobaR5Fv3QWmANTk6HpqVmKLEDy1xDsp54sw9sQqx
-lRrxt4NqZeaCZuuBwyn+pu5icIo01rguWU2P3r8WEbgbWh+JgGHvowTQEVz7jJVg
-yTduFzQLtAuskrz81gZ0Tmy8PnD2D2JRylXSmDwKeShFeV6r1eWBjmk25LlnpQ40
-TR4EmQINBF977m0BEAC8mpXeJ4uuAXsNOaunJVk6bP0D+P4SazaxTVTu6p3c68LN
-4F2zaxeVuagmf5Yk8u42EwwKfuhoaqX79n/hZYKRoZLsm1S6l+0jJFLVoUkUegXl
-oUdifxFSSXzZB2f06xVVDb7HS9KMdxL/26i9X893hwsNuOqRP3e5kFZFQSXbLuYn
-pFPrXhb/PVoJyEqb/iIvYsMgNMbKEN3bn6NBdq5FLy7T0Rr2v9U9yOqrWDvxMk1b
-WhmhoPeMkVOI6YBXckb3T4l7CH2I7N3PHBegv5m5zQ2zblSKrCzbKMe9mZ4yejXI
-u7tNA9ZRXPt+aLaaOC7pFZ+/skBqKVMn3y8X3dR7J6NulPxTP2sHpPYZ4RXuv0E4
-6WLgeoh08c4+VIyo5SCUoYJC1Sa6Fb/w7L48ZvkKACABrK+HZuaFNjn6ECWVpXqw
-aYfTMItm70MY9wcxzw70ugnUOja+YVrQv/qyBWtRgMKTIs+wz7EEpIepqe3C2WBq
-VJbTNXsGCi7O4KjKPyYbEGbTyh3BhyZA0btwvxqStlszrXXWkLInodfUt5DmVrqt
-qFKO8DQ0fTVhFTBxZ0nDpinSbNWjCx0FIm9fVf3QYT+dmirXbyqAGgvchpx67ar2
-Fzr71+jVdV1N57WGepteIeHhhT1XwuS0G5PMF1/yRbl3b8JsaQeoOKi9Us9aSwAR
-AQABtC9OVFBzZWMgU2VjdXJpdHkgUmVwb3J0aW5nIDxzZWN1cml0eUBudHBzZWMu
-b3JnPokCUwQTAQoAPhYhBOVyNdInZBKfpPL00X9SYI7Q5J12BQJfe+5tAhsDBQkJ
-ZgGABQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAAAoJEH9SYI7Q5J12L9cP+PuM4Dgr
-dizgNupWj7/szlv4FKeq10MNu1wM9+XByzp4OEnkgvJK5qm9O+/AvkwIsmh5G1qQ
-OnbvrhhNlUWzlBJ9McubUQYXQBUPiwS82QavGKhDW99IlV856+qpPhN30oQsqC0o
-U00V+NT7rQ9BU0CTgTfgdJ3CktIzjoQpUwXK8PwL3hd6cs7Z7n709RNrcP0omWHf
-CHc6FAHy5gH7p+vL+4raziuXS8ScKh2bY11NA0GhJ240ryRXKd1znv1wUdv9AwwG
-lfiqWtq8r1YQNKIik2slrkD8aIze0rw6JgKskc0vFpDV9iHkfw1pD3sX1W9TRID9
-nWqT9PLaVYDvInN4T+mPEnp4jFgp7B4mmr138GGtJFyTyzgcpvJv0oDsRxshBkEF
-MF3YYn5f6ZX3OZBlzclUH9yoNEnAn+GjaeZTW4V0qwjmnZD5g0OpVP1ljrxWlhAE
-v6D5yj7RIhg7Xdau/hlYZ83onRYT3isO5QDK9SGIKT2B2cclBzEuBhRXCxnF5uzL
-+ilpxUWx/0Xi8kOd1QBbfXMVCDkGSKIZQ1MatGFYQ7uGNnTNdbShTW15uXOmliC0
-kAchHHMT9rQ7i8tTe+ChgmRZ2pOdB77LyegHJ6Vp0FEH0ri4Qec2Qm6rV/nqaWTS
-GBFMncN0nu9LDJZS/S5Fdn1909Ze9sJM8YK5Ag0EX3vubQEQANseMvfX3Bxqdp2r
-BQOm4Pq8jYiRqlrUi8obgKwA6VC5IuyK+xyXd5O7Grq08DHMug1hxVsyBblTFe6m
-mmlZWrCX91441golgCGsgmGK52y37XOXEk5K7I9kHako+8xbFB4pwJqGICqTCu03
-XJsCHNrLYWB1x36FEEC80B5Riusp++i4sQPa5Bl1p+z5C8LNme0OLbDtRRNpNXDN
-q2r0GczT8uNIW5bTqXGqnv/MHnfQvcUHlapjVbsOMFFtbdrGyGPvKnwhjYg7xeeW
-IZ1id4uIfZrpm567Pailaf+bRkUVfY6tsoody/S8/4pY+XVr5DX+2Wd5rK9EGSj8
-CEy1JBTXKJ5AJa9sq2PGTzyXwA9/CgUhRRkqjyK7k1nqT6n/mD/uz2jMZIeodRes
-xl3BChnp3rs+zxb0I5kI6Kpu0zcjrMf/qA6eJciBmv9XlQwE9xG8tkzI1HB7rtds
-PFVYAP5byM387l/bX+Uev+RSSa8ZUI5HcNkiXY0MuljsTEok4f8Vl6cWjrXcZVJj
-+Grgo6x0pcRMwJAOODWZxlHAxelt9x3nKDdfO/pmRYx1/sx5MtgJs0Zht+xevksa
-vypPitSKjwGoDylvkGLc+d3+iKv6yrT4Nil6uxnSIsHhWW3GG4xWyDiy3vzwnlDj
-RnD3rZv8ntfhkq4JvaGdHpJD71tdABEBAAGJAjwEGAEKACYWIQTlcjXSJ2QSn6Ty
-9NF/UmCO0OSddgUCX3vubQIbDAUJCWYBgAAKCRB/UmCO0OSddleREACpajO7uKng
-tyYE+laputykGlYeReE0Hih4hL9om6hmkiYN1GSWP1i1g7Ce71pEZfT7Sm+O3xl2
-IU/GMUZz5etQk9wGgPKkRA5fEwq1KRJ1sXNBzSNKt8bvYa05BODWz4mAFsQYIiEZ
-RNMvvsSwg3O/cvDk6xH0Yp9fIZcMG2nbk+ktQlkd7SYwSbrvpenmv90b9diW4LUn
-zLBH6DmRdL+28PXUnbuZ4gbko8+8p8kU/ITNAh9dgTmiGDc8IiGmxPiSir2bnJiz
-k9mXyHub7yipMbgvRBra+oQMIibSk9SQA4qNNUiDGdYPRKZKL3yNyOTsb4F+qpql
-m2/HHqVXH9vcFYAFDjUkKXPikfoQGMvWf1v4Db9eNMKHS45Q2UgFTXNC0C+rwXxZ
-ouzdruGeBE5p733wBo3f8yNFabifPZr7nfvdb/+Z+TLMGRGXyH1hjR3eLjUjuF9p
-uIbfcoKjKgxsYDx/f74WKarH5qTZsbVzFoQHtCSKCKdMGS2JgzAr9pU3FSRjnTSO
-pRFr64n6/qGZ+0e0OjIAzgpDI9vDw8z2p20FNUlvLqfI4IaPrnpu9+msJLneC/Hz
-RlJoPBgR/KzCo65yKT0DBBWw/w1fMD0GidwJRnHy5EvF8oha6SaOQ/EhL7rT01lF
-+z7an3UhBdX8UAhjgSOt3rzVZiNxyp5N0w==
-=9YTT
+zSNOVFBzZWMgQ29udGFjdCA8Y29udGFjdEBudHBzZWMub3JnPsLBlgQTAQgAKQUC
+WK4Z3gIbAwUJBaOagAcLCQgHAwIBBhUIAgkKCwQWAgMBAh4BAheAACEJEAXZs3FH
+fHUoFiEE2j/fd0zHD6ZHKexFBdmzcUd8dSifKw//buxzCGY/M9EJSPfgUSJUH17y
+oPTCglHHDPcDtP0c5mNiqBgR3jFVIW4jPzzp641mhfeqP79CPar5VqS62TppDVHB
+qkAAvuT72RAVde9XQ9sb1CwRQKCtEooF79Xc9m2RM7RaJJstwb09jdkPc6aBgQBz
+GlXVTQs0Dp3cUq81vxJgeWUZt7JS+a41yMxQZkXvTgZEccuEuBg11JqY3G6bHXx3
+6bw/RunwyknlXnBrcUEc8gpJQOZfdC9nbka6ZbdGJbIkGnFK0X8OLeR59JP8glAK
+A1smtIK7jA7rwpHZ4tH2hmYOMLbjhCTRvEgSpkGx0FPZvQmRRGkY1AWhSk5QRRxj
+npAZuXOQ7VOzlkhtD9zT8pR9AGhuyToL9VDxNs6Sth1rf75wY+U5hvvfpFV5iGCt
+mGF9wjzfl1AFP3QDnd4hI81pUNAsHNNOzOoG594nPBWG1sHPENqMzP5nL2rZc+TU
+6chS2g57fUulezNdqVvi506/bZxF19iDlfJo8TH23vpUYFcbZDp1dhJot1Y31BIh
+eK2qE6+T8k+qng4g07gOB2DCoP4eprENgcungrlcfRCocB2wytALCrwWJ+GabP7x
+qvc2P4RUQ50jnBoNUMlwzNjywzl2cRQLOntGnWI7vdfzKSm+OHA0e9ClTlUP1YSI
+LFPD5RG+8CTiP/OZxK/NL05UUHNlYyBTZWN1cml0eSBSZXBvcnRpbmcgPHNlY3Vy
+aXR5QG50cHNlYy5vcmc+wsGWBBMBCABAAhsDBwsJCAcDAgEGFQgCCQoLBBYCAwEC
+HgECF4AWIQTaP993TMcPpkcp7EUF2bNxR3x1KAUCX3vnvAUJGzjeCQAKCRAF2bNx
+R3x1KMlEEACemiKp5P8PrgdQ4Q8CVzlqk2Ayca3o8VFYitRM32/Zxd4MlZDsMIh2
+WpXSuEfBFPdYvppljmKKC7UVO6fxuxX2FIIJDsG95MhkUAyD7huyMMEA02+15P/f
+b0CbVxKuUbMrpTkdUdSc1mE/Y+tWyqD05UgX+7eWQ7wAmNgG+nwmW+9uXYw5yiMB
+HsOpOyWYJlppyMD3O4gknov5XnopT6X3G87oDxc6iLzE93FLXWx5mIxE4O4pgLgA
+B2QGZw0plW45apNB/Ge3ESRojlDe36SK+RmJyWphWiORUKbvshAjf7bfDViKRbUQ
+knqV/0w3Our0MwbQDgGGilnyayVrO84EzJ2r3C2ltcvttlXf5r3BX4pB0hCFvSCj
+sNpJ87HNb/5DJMvx0J2SGaKnij944u616ggE4MdkTzi8FTzMJkAJuCMTidIY0iK3
+T3U8E33WLOFZS8EPsVe2trr2LgmhFbaxXcpnLXOwEwn6m1Qu18cPUDjpKzx8RhpG
+BkBJe+ZKukfSEvy4DY47FwdvlWjz47g5/HjtKsPst/TpzD+kh93lpxD9N3ekmPxL
+FgQKpW2M4i66w5VrATExe1pIxNBqCgaZzXo7vK8BKJucCtDEptfosALMRXmjdXXZ
+lLcdPrJVJTcw3fGCx1ukDs7Vs8TDPjOxAiEj4O8cFWW8MPTKKRNaW8LBlgQTAQgA
+KQUCVw8MswIbAwUJBaOagAcLCQgHAwIBBhUIAgkKCwQWAgMBAh4BAheAACEJEAXZ
+s3FHfHUoFiEE2j/fd0zHD6ZHKexFBdmzcUd8dSgiEhAAh6fFLUcdmUUNysgUJzRe
+pPkQ+E/ZOHUD6Jr3vD961ihGN+90O4zABaflBnqzYEyMhqbrNFCs7MKy8nkjk95b
+wOZ1Ouq/eSD6qVnmAcuKNZYEg6v8XQCDZEaM8Ngqc7FHYPgD1Fm4nXSn6pXA2hzn
+8lk/VaPN2C1/qNRWZFhOfu2NoSH4iOXLi2KUYQqDhlSZUhwTkdIoSJRfZ9TwAqPn
+XequPvissOOyPjghsdqsblktuQq8VhLqY0or0L8PA7g/VVkDGtN7FClFEPTEg67f
+rKAFGK7EobbQH8ulCkyGPopwwPnum9edKsyWeD2zLAoTQB4VAO+pR6jtZqwCEjYv
+r2zJ1zrIRkK7O9LL24MWImx4ut+nsp9cor0DQ/J2nVT20EMD1+qZSm6rqeF1Z7U+
+4nR+WQPfTcu1l6Uc2ydMi/yXNWEa2dbeUHjWK4S5XfyIzda7KTEtDpUHiBjL4nYQ
+kfAxLA40CeNOUAMIMIgGrpuQwYokNdlPHgKoKRfPTlXGvkN27deZgjVJAReH4+4S
+cGykbaphwYvG+iN8fnV6nJdIZmujPp+8HPbGzf30f5tDh0lf1OussESruO9ffad2
+I9HrUj9/bYFsYlHYb8dLXWznhgNUcoFzYy07EzmuRAR7inrVluFgDoDNrmKoKLUw
+FDY2pRWW5M9gDPeITEFhq6rCwXMEEAEIAB0WIQTu1KCJPcxwXbMJ4gLM8pxyOFIp
+BQUCX4zdMAAKCRDM8pxyOFIpBejsD/9W+L+xMd+PBUlwcmU8L6yWihdMutHYl2Fn
+9dpuRBIprc/3Ji3R/I3F1/K3Bqk3pWLRE/qPW8wOh/cU+4Ax30VnxtzTX/H1/Y3Z
+5shNQdexnlr67YuhmjC9bmPPHj2tDyr5N9a1GpCa/I3K8njMxPPFHmIwfpNfieP9
+t9bDfLqrYWfM5h8skKWgbI8Kq6zhru77nGdgvD983G4/ij/VVH+OabTMV0J4m+x+
+m7GIT+Sf8r0qFTCT5NUnhYqaKDqqRRD8qHiuGXvqEXhA2NTTIwnDLDuFmFtmUSK5
+n6ReGigRA0+LZvkR3c6HmD17cArByr/PDUT+90L/CTcCniQILpw9COnCUDkVZq/z
+GV5VVytH6i88wNSeVGMoiCjxgkP5h30W8mgjpnowpmIGNhJF2pQqLzaZqN7regKq
+f6tFYGCsd3+WTre0ierAfhoDXNX0D6FJG5DnmWS8+51OBgkYzYhVBhwl9SMaFX0f
+nxCMkK4WKPbOlo5e/24AOLJWZnRqSKzhNMJ8WDjj48DI+DW7yuKJPxO+2+2ZPxTI
+nkG/aCk9/xvXKQu8R5Nse37MBtriSWHe8pWZogs4ub0nrtEEc1aLMZMCQRteDeXX
+agslY49K9oyNOaRryUggsx5O9RTs43SONEG653OqcZjdbl+7y5h9zNFmSpII94WY
+1uTyLSViz8LBXAQQAQgABgUCVw8OCwAKCRAqfD42zCgtvsLKD/9ssv6W4gj9O5DZ
+f+ISfo+7AdrwmjN8ppvhy/699I3duGevKKQS7UEHY+Sjmf9YVWRzI1AGvJsjUsb/
+Btfa8zNijtP3PH6veTtlFOVeNP6OAjcUVgePBgHgMqu17m+EGGReH7a8ruBl7Z6w
+zIU60qHX27hPbZHm12W6X+EF7/+QevVurVdVYG3gzFM7J7Zbh05RWq0JUVimRk5z
+adrHRam+5yxPGsx1Hf5lzR2XIRoFxNzBh8eN9ilpbQ37GidTEyD4fOt3MKIEBGr1
+pFBnFc+ivxBsAMAI7a2bkku1v15d84zyjEAAgE17/AmYtqH2EIzMZ7I8a/sOIdLf
+aULT8uxr73yivC7YnGMPamL4KMmo0XHj/TjXop4nWIP4rma8G+yPYlk8PMpEHVyQ
+ZLcG1mtFmQKMtWSejiZfVNtcYzAVbrC1t0wF9b5x8VfJACVwV8JkGV8CwfabqEzo
+uIq1dh9Sm793bwG4LKd+W0DP5+QSYacpCEkgPeq4jVVZ+SEN+bCstEJwxW1WJ1p3
+rIBIGESpX+NKFAGlHl8i1fhrJ1foZr9Gx4L94GKrU4zgmB457I/t1k9iRpYVDfIC
+gY5sy6ZHcrkhLiOZhr199QWHv33Bd7u3Y5pqoLamA3bjPZEJk82p7ywMas0z4sg2
+kdzol7iC5OjBjySvgRFclkrSfKK9VsLBXAQQAQgABgUCVw8NmgAKCRBzIpFyYszB
+5d+2EACbTkI7XkF7wBI7pzEf8avOhwg0qKA3slX7oYqrnkaaCSMISllFm7U2tPTO
+JJQOtno4NfmURcvdMg3ozyYUeeBj+yUEYhM5PGm9j47pzcCkf8/2R9MpZ4oOCo9p
+KOuoWATG2wVMZEGh5aNs3Sq/f11ao9RSYEv1pxgy8dFBUQ+43v7WBImCWyU/jHWm
+PqewLOADzESZRriMGfaSXs1BbCfggmiV+EHe1RhbXCzdPDCIQugDTg+Z9kHrn44T
+TKXJJ3Fhord+klIwO06iR7poz8EO4ZiDqK1t2QBwtVAsUPd/RbO0lSuGQJ0OYDHF
+/6W3DJn1leGZ+4ozDWmrQEObt8ciVOYf+qPws/4qEdsw5E2r3ZsofBYCj4zqqJnt
+Zm4+3d98zpLIbZ/KdUAKPuDLZzxPbcvPXDM5RNUvFnnEjz6NJMcRP2CSUBxugndm
+qGQm5xXZ8DBNcfH73WtHyWedlxUaS/uY4VzvW+nA5OJLV/BvDzKTwdq4T4IrGX2F
+evg/Sl8PUWHeUlNyFMBa6c3gB90MjkznQFwPTX9WsEXkAIYSJSEdg+aHnhGiQXAu
+ETTqaEEiW76jtS6l8cJ5aFNXQa4GW5cZckJTlUHvkv8RTrTW8BUjgeAqazzQbTYD
+nhwEJms1pPASHPF4LXpmauffJ4qcKAsPTIfYSFptMMxRxn2JKs7BTQRXDwyzARAA
+0XTbXJBhJMaAOFvPWaw+dThy4RDlvV3hHiNJ9uIIROYnmiu4W6t+jrZsJ3WWdJqN
+ggY+XLnwVyRXWbmFXf8qViStoBfs62cDzmYndsfnl5ZcRdqrHJ+ErdblYU1Sc2Wd
+nfoJQY49VaSLNJMDvU9gRdC9dVdI9Z2YR+m7RNXNZbqtxdDMgMFZYBq3jdWNkutp
+QA9PKwco4qGpQdsWLpyGXZRHF6iz63WRWhAeYhLftUvoTXCISN+WTSuT4tC0pDfD
+fcdX7ber5icVg7Uh4aizFTenS4jlsbkE1I93oZCaPN3t8gah+iiCkIRVYpd0jy6R
+pZXIHLvRrgmdpZZ59Bkg3Xi4WtggJ5jg0wc8csC545M0wXlgktQN8w9RibR0woJo
+PUkh/dWiyXU/eEsQ84G1iMBHUGBDJ9g/vu/dnqhRiu/A+jx2JbeYuxvR1MmL6WXJ
+mQ8eHdJvnUQiS9nRqb8xQeSxoZlrBNw1QULk11fFMy0h3adOX3KXbnqZsPUvCkt5
+P3UyEJSQ4lnN5K/Acbd/5toENeB2kq1hLdJkHthrblnfkNtefY1fPHKtp56UqsuU
+caLUpvLijUntR4PSwAbhDNKXKdLJroSNV0WrrOH1Rn64apQlrPw0ZwUdIdjXQFCx
+dMsagL2TgwuULmYjQUP/noX0c4bmglqmcKelKrROdCEAEQEAAcLBdgQYAQgAIAIb
+DBYhBNo/33dMxw+mRynsRQXZs3FHfHUoBQJfe+dGAAoJEAXZs3FHfHUoEq8P/35S
+lCVQIwWv9x7Bg3rpJR1WO5VDX//+UWWP5i2+fF8LGgkwNNno63VSSsJmla8A6IAT
+5pAisixCJuXJrQ2RWUiVgCOvUIOgczQFuv83T+mnbFh6Y9Ic9Ag4p5sLRDiwofce
+B+lNIOlmjAdg1KhFbBEczyRWqho5khfKSsxMOtJ1LhcGwEhWF3DeHjJgGdb1SuAw
+pxiLpa/DgASLYXhEFfayYxlT4la+SkpGJXzKgEZ7vc5GB0FKAD0/Jz9Yq1fBoR0j
+6pyEhqa4p1okUTP8YxsNHmw9gUcwLqs44JS/AhTJ1pghdha0m3h4pUV7k1LpqXbC
+qKPl+edwNLotN9I7WlfcEJuBAudP46f9mq29N0cp3ATUmuZsioPkr+u+LDK7uSyl
+AJaJ3wG+RSGjTiV4N/AlN6W4syT8PAT/cfjWsdboKD0Df8lVx/PUHsmeLc1vfNoa
+wo7KwjNVaksBiXvGrXgMhTs3Koub5rMpyMB8OXsFGPwho62yr6euMW1zrxmlobaR
+5Fv3QWmANTk6HpqVmKLEDy1xDsp54sw9sQqxlRrxt4NqZeaCZuuBwyn+pu5icIo0
+1rguWU2P3r8WEbgbWh+JgGHvowTQEVz7jJVgyTduFzQLtAuskrz81gZ0Tmy8PnD2
+D2JRylXSmDwKeShFeV6r1eWBjmk25LlnpQ40TR4EwsF8BBgBCAAPBQJXDwyzAhsM
+BQkFo5qAACEJEAXZs3FHfHUoFiEE2j/fd0zHD6ZHKexFBdmzcUd8dShgSA//ZLX9
+7rs//cjKby8aqMxMlhaQzsCCqpCBd2Pi/X0u2BFccO2+QxKkuK57p5jNWL6WgetU
+z6x/HSP8UHXP9uB6QGu4EAkz4XM/FQwXtq8Uv8HxJgnF+c3jgIWjb4BhCkhDLm9X
+8+tTstQkQLxjkCjhuZVYjYmnMizYDw+XElBYYPjNlTKjdPNukB7QknI/bvhEO8Va
+3NbWYt+nI7W2lirDFLwvFpI1SjcWgigxqq32vRAOLKetZM7WkWeSDqodNuRYlaz4
+80pbRTePpIyCnL8ist38ueuyQ2MXqfWlm688GzDR3JOVdKAnxsGj514+BpS8VAts
+DZJLxNBS/7uNmHV4Haawd+lZywGLzzzuQE+ZaBENw7axgvyLh2f4i8+FBNktgfqA
+yqrWMutLB90Otiqc9J0nqkZlfbA9PNNNmOCtJaU3MoX83Qc0JtW8zxjrnBGtHJ+p
+TLkU0LG0ENsqpN7TjG8wfX6o09KxblmsIFiuuycKjaqumoy/qqR4q3ugJLWbS3MU
+oom1Hs+8gwCY9ZhbRbVfZBmMPmzhcUhJxj65sovnCFQXk8XBGqqcDj2ItkgmmAZB
+PrVUZB5Fg1pwm9TXAB2Juy6vOqiKn5EAWZ8RR6riZB/ARX0mMT3y/p7B2s7Ddtf3
+qwN0WCUjvW4JVkmp0yHHi2lH9Uet/W2EB6yjrhbGwU0EX3vubQEQALyald4ni64B
+ew05q6clWTps/QP4/hJrNrFNVO7qndzrws3gXbNrF5W5qCZ/liTy7jYTDAp+6Ghq
+pfv2f+FlgpGhkuybVLqX7SMkUtWhSRR6BeWhR2J/EVJJfNkHZ/TrFVUNvsdL0ox3
+Ev/bqL1fz3eHCw246pE/d7mQVkVBJdsu5iekU+teFv89WgnISpv+Ii9iwyA0xsoQ
+3dufo0F2rkUvLtPRGva/1T3I6qtYO/EyTVtaGaGg94yRU4jpgFdyRvdPiXsIfYjs
+3c8cF6C/mbnNDbNuVIqsLNsox72ZnjJ6Nci7u00D1lFc+35otpo4LukVn7+yQGop
+UyffLxfd1Hsno26U/FM/awek9hnhFe6/QTjpYuB6iHTxzj5UjKjlIJShgkLVJroV
+v/Dsvjxm+QoAIAGsr4dm5oU2OfoQJZWlerBph9Mwi2bvQxj3BzHPDvS6CdQ6Nr5h
+WtC/+rIFa1GAwpMiz7DPsQSkh6mp7cLZYGpUltM1ewYKLs7gqMo/JhsQZtPKHcGH
+JkDRu3C/GpK2WzOtddaQsieh19S3kOZWuq2oUo7wNDR9NWEVMHFnScOmKdJs1aML
+HQUib19V/dBhP52aKtdvKoAaC9yGnHrtqvYXOvvX6NV1XU3ntYZ6m14h4eGFPVfC
+5LQbk8wXX/JFuXdvwmxpB6g4qL1Sz1pLABEBAAHNL05UUHNlYyBTZWN1cml0eSBS
+ZXBvcnRpbmcgPHNlY3VyaXR5QG50cHNlYy5vcmc+wsGUBBMBCgA+AhsDBQsJCAcC
+BhUKCQgLAgQWAgMBAh4BAheAFiEE5XI10idkEp+k8vTRf1JgjtDknXYFAmpsItUF
+CQ6ym2gACgkQf1JgjtDknXbVcBAAq6Pow0MyNgx1r9PJxYR0MTitzoP+GEylGvpk
+QyPOpZEFvYqnpHh7TNv6k3wKMYRXcPSE+ksr2ZuC6i2EHtCd/xF9ufLqty8HIzqJ
+tZbcXC1B0u9h6IXg2vDbdti965NW+xHP33bzpC0Un6w41iAU0kYMmDQIWQg4fKIf
+8QrFi9Hz0nWw9UXkGZUPBhhS+6HjSFQdztsWSbgy41787opMijAgfctubevs5M95
+fj0CwKX8H0viXF1rHKnTJmgzUGUj+scwJVoUxbBHa4IPEGysxHl4i/peUY5dnEtL
+8kQX2U2Xa/f2IavdYrz3tPOhPsAvWRJmOBLXadnA04ItidLxM7g84SGOvwCjX5cG
+kVCXYF+sy3/2VlLZ4q61RQoZtE+DJvbGjxaL/QahRMhQhtWjtSbDN/P1MIsJuLj9
+jP72Pgqo+n8FaRxq4IEu4J5undkZ98sVSL9/YHgP3AuKAkwpTX2pH6deQMxyvlhB
+FkkYTUfZyTeLfE3cftqsGeg570A8wau88mremZv69arnG/9370km/UI/lIM8WxLX
+CaBVK5ALU3lV53G2fVBT+9f35wY7tSFglV98J1RscOPfK/KUTghxpZzyc8BHHQwU
+FqAMx98793EKi8Ijimir420wVn19jJpOZNq3ooltrIbUuVYeV4gVcWxeorGw/AFJ
+dCNyl1PCwZMEEwEKAD4WIQTlcjXSJ2QSn6Ty9NF/UmCO0OSddgUCX3vubQIbAwUJ
+CWYBgAULCQgHAgYVCgkICwIEFgIDAQIeAQIXgAAKCRB/UmCO0OSddi/XD/j7jOA4
+K3Ys4DbqVo+/7M5b+BSnqtdDDbtcDPflwcs6eDhJ5ILySuapvTvvwL5MCLJoeRta
+kDp2764YTZVFs5QSfTHLm1EGF0AVD4sEvNkGrxioQ1vfSJVfOevqqT4Td9KELKgt
+KFNNFfjU+60PQVNAk4E34HSdwpLSM46EKVMFyvD8C94XenLO2e5+9PUTa3D9KJlh
+3wh3OhQB8uYB+6fry/uK2s4rl0vEnCodm2NdTQNBoSduNK8kVyndc579cFHb/QMM
+BpX4qlravK9WEDSiIpNrJa5A/GiM3tK8OiYCrJHNLxaQ1fYh5H8NaQ97F9VvU0SA
+/Z1qk/Ty2lWA7yJzeE/pjxJ6eIxYKeweJpq9d/BhrSRck8s4HKbyb9KA7EcbIQZB
+BTBd2GJ+X+mV9zmQZc3JVB/cqDRJwJ/ho2nmU1uFdKsI5p2Q+YNDqVT9ZY68VpYQ
+BL+g+co+0SIYO13Wrv4ZWGfN6J0WE94rDuUAyvUhiCk9gdnHJQcxLgYUVwsZxebs
+y/opacVFsf9F4vJDndUAW31zFQg5BkiiGUNTGrRhWEO7hjZ0zXW0oU1teblzppYg
+tJAHIRxzE/a0O4vLU3vgoYJkWdqTnQe+y8noByeladBRB9K4uEHnNkJuq1f56mlk
+0hgRTJ3DdJ7vSwyWUv0uRXZ9fdPWXvbCTPGCwsFzBBABCAAdFiEE7tSgiT3McF2z
+CeICzPKccjhSKQUFAl+M3UgACgkQzPKccjhSKQWBWg/+OT/MQmEB/kBZJpAJZzUh
+hbEWMtq7U+5TMn+DKUpKRmadSl1ZJpg++l6Bo/b4oU5ICsBDhmeJ/PmPOT8EnRLH
+7KmVUIMetbQYn0TYpllxbnPueT4a2Qv5YzwjxLBv1dprp1m4vU9VI8B+BTG34Vrk
+kX/vBla8BQoD0qfIA3afF5ZqkpQy2t/PICkojOHAFPw4G9i1ENDlgOPFl39/C6bJ
+Wr4/iFyOXMA1da8CDOlu57lZdJMl54oc59qtBdTInRsYKXdZB/h2EQqU39yqd8bO
+OdxLzcyjXk7VUX3WYGf6N8NqEpg3K49N60gBl47/Wx53kKGcMQ9MToyhTgVRiaqv
+4I2kBi9HF0wdud3O5FwLgJUy87Pj69mCliyxZJVNLXsSElOinLQdIKgK4HDs7md6
+8abgJclHAU3wwBDO79KEarqXbpKGugLqoWluK70/DU5Emzg9cA7Hg8ZA4ClD/lA6
+i9CTPK0m7KZhR2EZBQoS3DGvK8Ay69/POgJMDP3cDXqkl3JZHrw+O6urcwhnLJgq
+4mcM0BQfc80ShGqvAzcHrBrYs0pLiMVKCX2/4QsWH+ga5ORPb/c8sw3QNdTl/GwH
+zpqPQp+JkMIMzYzZk7uI8roeS0yZN1mFqp4AatvOMJhLS39e0bjvwh7grRkbhLDm
+7qyMXeXdM1PfgDotE/DsNh7OwU0EX3vubQEQANseMvfX3Bxqdp2rBQOm4Pq8jYiR
+qlrUi8obgKwA6VC5IuyK+xyXd5O7Grq08DHMug1hxVsyBblTFe6mmmlZWrCX9144
+1golgCGsgmGK52y37XOXEk5K7I9kHako+8xbFB4pwJqGICqTCu03XJsCHNrLYWB1
+x36FEEC80B5Riusp++i4sQPa5Bl1p+z5C8LNme0OLbDtRRNpNXDNq2r0GczT8uNI
+W5bTqXGqnv/MHnfQvcUHlapjVbsOMFFtbdrGyGPvKnwhjYg7xeeWIZ1id4uIfZrp
+m567Pailaf+bRkUVfY6tsoody/S8/4pY+XVr5DX+2Wd5rK9EGSj8CEy1JBTXKJ5A
+Ja9sq2PGTzyXwA9/CgUhRRkqjyK7k1nqT6n/mD/uz2jMZIeodResxl3BChnp3rs+
+zxb0I5kI6Kpu0zcjrMf/qA6eJciBmv9XlQwE9xG8tkzI1HB7rtdsPFVYAP5byM38
+7l/bX+Uev+RSSa8ZUI5HcNkiXY0MuljsTEok4f8Vl6cWjrXcZVJj+Grgo6x0pcRM
+wJAOODWZxlHAxelt9x3nKDdfO/pmRYx1/sx5MtgJs0Zht+xevksavypPitSKjwGo
+DylvkGLc+d3+iKv6yrT4Nil6uxnSIsHhWW3GG4xWyDiy3vzwnlDjRnD3rZv8ntfh
+kq4JvaGdHpJD71tdABEBAAHCwXwEGAEKACYCGwwWIQTlcjXSJ2QSn6Ty9NF/UmCO
+0OSddgUCamwjWAUJDrKb6wAKCRB/UmCO0OSddto8D/98Kx9Lwe/cfeRI0VMrGSO9
+SprFrrEObJ1exZqFWslJ8ASp0dMUjRUjkpBBf0CzNIQerAX1Zpr3xk+46oV2HiqS
+kL61bkvDX84GYmbopBYCmWPuz/WGOb7TSAsMB1qxKIkxfO3tv7H4SnCp9TzpqvK1
+jUlPX+F/9u5AaA723//q6pkOSZ7r7c9wuRt0ayVXfB2pepkamR954kPxOOQ5dSpZ
+tS4t6a3A7ZuBhLS0qjd8gv3ryAXt+WbJt/KYHGvPmXMlDzkjhJh7WT/OY8JrG6YM
+2Mlnd/qRVxz0XfM+xTYCIISWtkhA9SyGqZDiHAxWyS8FiNGyb2z3W4Pb48D9C6ql
+VMHgMsA5sf2hTuN3PAqJzc+0OpupaSFusxWqtW0HPUIBDFLi9fJy0fGNEj5GUup7
+gfd0937e8nQF+mhDJdd2iOaTL7ABWATaTW6L5RLx+S7JTihaDs8dEgIO5BXYZGuE
+smrJMjTu5aGXEvsgey09izzgni7dCrnzwtodAy+aBqwY3gdewC9qHaSxNoQA36B9
+im4UAHPPf+/LucGHleVdh4WSxepOIdPsHs/p3spAnYGJFMOyrwff1ritLkae0YXQ
+BrI6iaV/ePMWqEm/HdYcvo1jmMWVSD0fgYmzcN2Wu84O3lsPo4GEeA51xEsJjV4o
+cnXRHp4ap+uIxy14G8O5T8LBfAQYAQoAJhYhBOVyNdInZBKfpPL00X9SYI7Q5J12
+BQJfe+5tAhsMBQkJZgGAAAoJEH9SYI7Q5J12V5EQAKlqM7u4qeC3JgT6Vqm63KQa
+Vh5F4TQeKHiEv2ibqGaSJg3UZJY/WLWDsJ7vWkRl9PtKb47fGXYhT8YxRnPl61CT
+3AaA8qREDl8TCrUpEnWxc0HNI0q3xu9hrTkE4NbPiYAWxBgiIRlE0y++xLCDc79y
+8OTrEfRin18hlwwbaduT6S1CWR3tJjBJuu+l6ea/3Rv12JbgtSfMsEfoOZF0v7bw
+9dSdu5niBuSjz7ynyRT8hM0CH12BOaIYNzwiIabE+JKKvZucmLOT2ZfIe5vvKKkx
+uC9EGtr6hAwiJtKT1JADio01SIMZ1g9EpkovfI3I5OxvgX6qmqWbb8cepVcf29wV
+gAUONSQpc+KR+hAYy9Z/W/gNv140wodLjlDZSAVNc0LQL6vBfFmi7N2u4Z4ETmnv
+ffAGjd/zI0VpuJ89mvud+91v/5n5MswZEZfIfWGNHd4uNSO4X2m4ht9ygqMqDGxg
+PH9/vhYpqsfmpNmxtXMWhAe0JIoIp0wZLYmDMCv2lTcVJGOdNI6lEWvrifr+oZn7
+R7Q6MgDOCkMj28PDzPanbQU1SW8up8jgho+uem736awkud4L8fNGUmg8GBH8rMKj
+rnIpPQMEFbD/DV8wPQaJ3AlGcfLkS8XyiFrpJo5D8SEvutPTWUX7PtqfdSEF1fxQ
+CGOBI63evNVmI3HKnk3T
+=KrYV
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/ntpsec.spec b/ntpsec.spec
index 3e1584f..bf6a5b3 100644
--- a/ntpsec.spec
+++ b/ntpsec.spec
@@ -7,6 +7,10 @@ License:        NTP AND BSD-2-Clause AND BSD-3-Clause AND BSD-4-Clause AND ISC A
 URL:            https://www.ntpsec.org/
 Source0:        https://ftp.ntpsec.org/pub/releases/ntpsec-%{version}.tar.gz
 Source1:        https://ftp.ntpsec.org/pub/releases/ntpsec-%{version}.tar.gz.asc
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - DA3FDF774CC70FA64729EC4505D9B371477C7528 E57235D22764129FA4F2F4D17F52608ED0E49D76 | sq keyring filter --experimental --cert DA3FDF774CC70FA64729EC4505D9B371477C7528 --cert E57235D22764129FA4F2F4D17F52608ED0E49D76 --output update.pgp
+#   sq keyring merge --output ntpsec.gpg.pub.asc.new ntpsec.gpg.pub.asc update.pgp
+#   mv ntpsec.gpg.pub.asc.new ntpsec.gpg.pub.asc
 Source2:        https://ftp.ntpsec.org/pub/releases/ntpsec.gpg.pub.asc
 Source3:        ntp.conf
 
-- 
2.56.0


